framework/composer.json

```diff
@@ -65,7 +65,7 @@
         "ezyang/htmlpurifier": "~4.6",
         "cebe/markdown": "~1.0.0 | ~1.1.0",
         "bower-asset/jquery": "2.2.*@stable | 2.1.*@stable | 1.11.*@stable | 1.12.*@stable",
-        "bower-asset/jquery.inputmask": "~3.2.2",
+        "bower-asset/jquery.inputmask": "~3.3.3",
```

Comment:
I tend to review the changes of my dependencies to make sure nothing breaks.
A `composer update yiisoft/yii2` will fail and not update to 2.0.11 with this requirement because it will only update yiisoft/yii2 and no other packages. It should be possible to upgrade yii without touching any other packges. The only exception I see here are security issues that force the need of bumping a dependency version.